src/frontend/components/authentication/Login.jsx

```diff
@@ -20,8 +20,9 @@ const Login = () => {
             
                 const response = await axios.post("/api/auth/login",userData)
                 if(response.status === 200){
-                    localStorage.setItem("JWT_TOKEN_MOZI",response.data.encodedToken)
-                    localStorage.setItem("USER_PROFILE_MOZI",JSON.stringify(response.data.foundUser))
+                    localStorage.setItem("mozi",JSON.stringify({"JWT_TOKEN_MOZI":response.data.encodedToken, "USER_PROFILE_MOZI":response.data.foundUser.firstName}))
+                    // localStorage.setItem("JWT_TOKEN_MOZI",response.data.encodedToken)
+                    // localStorage.setItem("USER_PROFILE_MOZI",JSON.stringify(response.data.foundUser))
```

Comment:
Here also, if not required, remove the comments

Reply:
Commented code is removed